docs/configuration/secrets/troubleshoot.md

```diff
@@ -0,0 +1,35 @@
+# Troubleshooting
+
+The sections below include information to help you troubleshoot issues that you may come across while configuring the secrets used by Codacy.
+
+If the information provided on this page is not enough to solve your issue, contact [support@codacy.com](mailto:support@codacy.com) providing all the information that you were able to collect while following the troubleshooting instructions.
+
+## Lost/Changed initially generated secrets
+
+While trying to open Codacy,
+you were shown an error message stating that,
+the secret on the database and the one in your configuration file are different.
+
+### Steps
+
+1. Get the latest logs from the API service
+
+    a. Get logs and filter by the prefix of the message
+
+        ```bash
+        bash <(curl -fsSL https://raw.githubusercontent.com/codacy/chart/master/docs/configuration/secrets/extract-codacy-secrets.sh) \
+            -n <namespace>
+        ```
+
+        You can also download the script [extract-codacy-secrets.sh](extract-codacy-secrets.sh) to run it manually.
+
+2. Copy the value and update it in your values yaml file
+
+3. Restart the application with the new configuration
```

Comment:
For consistency we can use exactly the same text here as we are using in the multiple existing places in the documentation (see step 4 of [this page](https://codacy.github.io/chart/configuration/integrations/github-cloud/), for instance).

If later we decide to extract and centralize this step somewhere else, it makes it easier to do the refactor if we already know that the instruction is exactly the same everywhere.